source/Calamari.Tests/Fixtures/Util/StringExtensionsFixture.cs

```diff
@@ -27,5 +28,19 @@ public void NullValueShouldThrowException()
             Action action = () => "foo".ContainsIgnoreCase(null);
             action.Should().Throw<ArgumentNullException>();
         }
+
+        // This method has some weird legacy issues which we now rely on
+        [TestCase(@"C:\Path\To\File1.txt", @"C:\Path\", @"To/File1.txt")]
+        [TestCase(@"C:\Path\To\File2.txt", @"C:\Path", @"To/File2.txt")]
+        [TestCase(@"C:\Path\To\File3 With Spaces.txt", @"C:\Path", @"To/File3 With Spaces.txt")]
+        //[TestCase(@"C:/Path/To/File4.txt", @"C:/Path/", @"To/File4.txt")]
```

Comment:
Whats the go with this test case? Can we uncomment or delete?

Reply:
I was a bit confused here with the `/` `\` changing in assertions in the tests 1-3. Are we doing this to check that both types of slashes have the same result?

Reply:
It's a weird case which produces `../To/File4.txt` which is an unexpected output. I'd prefer to delete.

This method returns some unexpected results when it comes to non-windows style paths. I'm uncomfortable changing the behaviour given we rely on it elsewhere.

Reply:
I've removed that comment